ios/LibTorchvision.podspec

```diff
@@ -1,8 +1,8 @@
-pytorch_version = '1.10.0'
+pytorch_version = '1.11.0-SNAPSHOT'
```

Comment:
The iOS setup job actually installs the nightly versions, so I guess marking it like that makes sense? On the other hand, `-SNAPSHOT` is Java lingo. Perhaps replace with `-nightly`?